observer/doppler.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+
+def calculate_doppler_shift(frequency, distance=None, angular_velocity=None):
+    """ Calculates doppler effect frequency shift and returns new frequency.
+    """
+
+    #  THIS IS NOT IMPLEMENTED YET !!!
```

Comment:
I suggest we move, TODOs and comments to issues in order to easily track them and keep the code clean.